menus/sync-settings.cson

```diff
@@ -8,6 +8,7 @@
         { 'label': 'Backup', 'command': 'sync-settings:backup' }
         { 'label': 'Restore', 'command': 'sync-settings:restore' }
         { 'label': 'View backup', 'command': 'sync-settings:view-backup' }
+        { 'label': 'Check for updated backup', 'command': 'sync-settings:check-backup' }
```

Comment:
Atom uses `Check for Updates` but besides checking the gist hash, for us that would also involve checking if files (config, snippets, etc.) have changed since last backup. Future enhancement ... ? :bulb: